pkg/mkimage-raw-efi/make-raw

```diff
@@ -220,7 +262,10 @@ calc_image_size() {
               SIZE=$((SIZE + $(calc_rootfs_part_size)))
               ;;
           imgb)
-              SIZE=$((SIZE + $(calc_rootfs_part_size)))
```

Comment:
I don't think we need to allow a different size for IMGB, it should be the same as IMGA...

Reply:
@rene it will be non-zero only for evaluation so no need to check